content/engine/install/debian.md

```diff
@@ -18,12 +18,23 @@ To get started with Docker Engine on Debian, make sure you
 
 ## Prerequisites
 
-> **Note**
+### Firewall limitations
+
+> **Warning**
 >
-> If you use ufw or firewalld to manage firewall settings, be aware that
-> when you expose container ports using Docker, these ports bypass your
-> firewall rules. For more information, refer to
-> [Docker and ufw](../../network/packet-filtering-firewalls.md#docker-and-ufw).
+> Before you install Docker, make sure you consider the following
+> security implications and firewall incompatibilities.
+{ .warning }
+
+- If you use ufw or firewalld to manage firewall settings, be aware that
```

Comment:
I don't think Docker bypasses firewalld rules created by end-users. (but @robmry can keep me honest.)

@dvdksn Do you remember why you put it here?

Reply:
I think it was due to this comment: https://github.com/docker/docs/pull/17176#discussion_r1212603437

Reply:
We put bridges into a "docker" firewalld zone, which amounts to "ACCEPT" for everything (then we manage the actual policies with the usual iptables rules).

So, "we mess with your rules" is probably a fair warning. The link is to a `Docker and ufw` section, but it's adjacent to `Integration with firewalld`. We could probably add some detail to both of those. But, that text isn't changed by this PR, so I think it's ok as it is for now - and worth getting the `nft` warning in.

Reply:
(Oh, oops - I didn't mean for this reply to show up as a separate comment below as well. Thanks GitHub!)